### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by tabascopepper** [community/community#134435] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

After almost 2 days I do not received response to my GitHub Support Ticket. I see that many people are waiting for an answer for months. But i need to resolve this problem with billings as fast as possible

Why this is happening? Short staffed or some bugs maybe?

**Comment by wiizzoux** [comment]
Same problem. No response since May 7th (88 days!)

**Reply by mferitsimsek** [reply]
Same since May 30 . Github support team response time cannot be 82 days. This is a terrible situation!

**Comment by AmineDerbal** [comment]
Same for me no response for more than a month

**Comment by SomethingNew71** [comment]
I am in the same boat no response for a week, not quite as long as you but this is outrageous. If they plan to flag accounts to disable them in this way they need a fast avenue to resolve it after the issues have been fixed. I had an API key compromised and 5 issues created. Now my account and all its projects are held hostage and I cant deploy things for about a week now. No response on a ticket I opened.
